README.md

```diff
@@ -1,7 +1,41 @@
 # help-center-content
+
 Repository assembling VTEX's Help Center content
 
+## In this repository
+
+You will find the following folders in this repository:
+
+-TBD
+
+## Contributing with Help Center Portal documentation
+
+We're so glad you're here! Thanks for being interested.
+
+### How do I add a download file to my article?
+
+1. Open a branch in the [help-portal-content](https://github.com/vtexdocs/help-center-content) repository.
+2. Copy the file you want to make available for download to the same directory as the article where it will be inserted. For instance, to allow the user to download a spreadsheet within the article **[Payment Provider List](https://github.com/vtexdocs/help-center-content/blob/b454803127cc490362c16921ceee023a5d182409/docs/pt/tutorials/Payments/Payment%20Settings/list-of-payment-providers-by-country.md)**, you need to save it in the directory [vtexdocs/help-center-content/docs/pt/tutorials/Payments/Payment Settings](https://github.com/vtexdocs/help-center-content/docs/pt/tutorials/Payments/Payment_Settings/).
+3. Go to the [branches page](https://github.com/vtexdocs/help-center-content/branches), and under **Your branches** section, click on the name of the branch you created.
+
+![Branches_page](https://raw.githubusercontent.com/vtexdocs/help-center-content/readme-download-illustration-1.png)
```

Comment:
Images are not loading. I think they are missing `/tree/{branch}` maybe? Or will they load once we merge?

Reply:
That's right. Once the merge is done, it should appear. But I also took the opportunity to update with `/main`